test/controllers/api/stories_controller_test.rb

```diff
@@ -35,8 +35,30 @@ class << @controller; attr_accessor :prx_auth_token; end
       Story.find(id).account_id.must_equal account.id
     end
 
+    it 'can create a new story for a series' do
+      stub_request(:post, 'https://id.prx.org/token').to_return(status: 500)
```

Comment:
oh never mind, just saw the comment from earlier (might be worth reinstating?) 
`# feeder calls do't need to work, but webmock needs a response defined`